maintainers/github-teams.json

```diff
@@ -146,7 +144,6 @@
       "peterhoeg": 722550
     },
     "members": {
-      "Br1ght0ne": 12615679,
```

Comment:
As the maintainer for the crystal-lang team, @peterhoeg, please implement the removal of `Br1ght0ne` from the team.